bin/opm

```diff
@@ -183,30 +197,29 @@ sub do_build ($) {
 
     my $account;
     if (!$server_build) {
-        my ($rcfile, $data) = get_rc_file();
-        my $default_sec = $data->{default};
-        $account = delete $default_sec->{github_account};
+        my ($rc_file, $rc_data) = get_rc_file();
+        my $rc_default_sec = $rc_data->{default};
```

Comment:
Would you mind if I do a PR first that standardizes the variable names?  This was a bit of a headache to unwind, because `$data` and `$default_sec` would be recycled from the rc file to the ini file.